docs/mkdocs/index.md

```diff
@@ -1,11 +1,14 @@
 # Home
 
-Kernel Module Management optionally builds and runs DriverContainers on Kubernetes clusters.
+## Overview
 
-## Getting started
+Kernel Module Management also known as KMM is a Kubernetes operator which provides a simple way to manage, build, sign or deploy out-of-tree kernel modules, drivers or device plugins on selected nodes of an existing Kubernetes cluster.
 
-Install the bleeding edge version of the KMM Operator by running the following command:
+Once installed we will get a Deployment object consisting of a ReplicaSet with a pod acting as a controller manager in the new Namespace `kmm-operator-system`.
```

Comment:
I would remove this. If installed with OLM, the namespace is configurable, and the Deployment / ReplicaSet bits are common to most operators.
```suggestion
```